### PR TITLE
Migrate ServiceNow modules to Collection

### PR DESCRIPTION
**Pull request by willtome**
##### SUMMARY
Update BOTMETA with location of snow_record.py and snow_record_find.py

##### COMPONENT NAME
`/lib/ansible/modules/notification/snow_record.py`
`/lib/ansible/modules/notification/snow_record_find.py`

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -369,6 +369,10 @@ files:
   $modules/notification/rocketchat.py:
     ignored: ramondelafuente
     maintainers: Deepakkothandan
+  $modules/notification/snow_record.py:
+    migrated_to: servicenow.servicenow
```
Comment (gundalow) [inline .github/BOTMETA.yml:804]: Thanks, though I don't see servicenow.servicenow on Galaxy. Have they been published yet?

Comment (willtome) [reply .github/BOTMETA.yml:804]: https://galaxy.ansible.com/servicenow/servicenow

Comment (gundalow) [reply .github/BOTMETA.yml:804]: Thanks.
Can you please add module_utils and the inventory file to botmeta.yml as well.

**Comment by Akasurde** [comment]
cc @n3pjk @garbled1

File: .github/BOTMETA.yml (hunk 1 above)
Comment (willtome) [reply .github/BOTMETA.yml:804]: Added. Inventory was never part of core (#63912) so it will just be new the collection.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Perfect, thank you.

**Pull request merged**